redash/monitor.py

```diff
@@ -30,15 +30,23 @@ def get_object_counts():
 
 
 def get_queues_status():
-    return {queue.name: {"size": len(queue)} for queue in Queue.all(connection=rq_redis_connection)}
+    return {
+        queue.name: {"size": len(queue)}
+        for queue in Queue.all(connection=rq_redis_connection)
+    }
 
 
 def get_db_sizes():
+    schema = get_schema()
+    query_results = "query_results"
+    if schema:
+        query_results = ".".join([schema, query_results])
+
     database_metrics = []
     queries = [
         [
             "Query Results Size",
-            "select pg_total_relation_size('query_results') as size from (select 1) as a",
+            f"select pg_total_relation_size('{query_results}') as size from (select 1) as a",
```

Comment:
This doesn't respect the schema on engine since it is a nested closure of the `pg_total_relation_size` of the query. This is the only place I had to manually prefix.